profiles/VP_ANDROID_15.json

```diff
@@ -0,0 +1,205 @@
+                        "bufferFeatures": []
+                    }
+                },
+                "VK_FORMAT_A4R4G4B4_UNORM_PACK16_EXT": {
+                    "VkFormatProperties": {
+                        "linearTilingFeatures": ["VK_FORMAT_FEATURE_COLOR_ATTACHMENT_BIT", "VK_FORMAT_FEATURE_BLIT_DST_BIT", "VK_FORMAT_FEATURE_TRANSFER_SRC_BIT", "VK_FORMAT_FEATURE_TRANSFER_DST_BIT"],
+                        "optimalTilingFeatures": ["VK_FORMAT_FEATURE_SAMPLED_IMAGE_BIT", "VK_FORMAT_FEATURE_COLOR_ATTACHMENT_BIT", "VK_FORMAT_FEATURE_BLIT_SRC_BIT", "VK_FORMAT_FEATURE_BLIT_DST_BIT", "VK_FORMAT_FEATURE_TRANSFER_SRC_BIT", "VK_FORMAT_FEATURE_TRANSFER_DST_BIT"],
+                        "bufferFeatures": []
+                    }
+                }
+            }
+        },
+        "DepthClipControl": {
+            "extensions": {
+                "VK_EXT_depth_clip_control": 1
+            },
+        },
+        "maxGeometryTotalOutputComponents": {
+            "features": {
+                "properties": {
+                    "VkPhysicalDeviceProperties": {
+                        "limits": {
+                            "maxGeometryTotalOutputComponents": 1028,
+                        }
+                    }
+                }
+            },
+        },
+        "primitivesGeneratedQuery": {
+            "extensions": {
+                "VK_EXT_primitives_generated_query": 1
+            },
+            "features": {
+                "VkPhysicalDevicePrimitivesGeneratedQueryFeaturesEXT": {
+                    "primitivesGeneratedQuery": true
+                }
+            }
+        },
+        "pipelineStatisticsQuery": {
+            "features": {
+                "VkPhysicalDeviceFeatures": {
+                    "pipelineStatisticsQuery": true
+                }
+            }
+        },
+        "androidYUVtarget": {
+            "extensions": {
+                "VK_ANDROID_external_format_resolve": 1
+            }
+        },
+        "generalYUVtarget": {
+            "extensions": {
+                "VK_KHR_ycbcr_rendertarget": 1
+            }
+        },
+        "swBresenhamLines": {
+            "extensions": {
+                "VK_EXT_line_rasterization": 1
+            },
+            "features": {
+                "VkPhysicalDeviceLineRasterizationFeaturesEXT": {
+                    "bresenhamLines": true
+                }
+            }
+        },
+        "hwBresenhamLines": {
+            "extensions": {
+                "VK_IMG_relaxed_line_rasterization": 1
+            },
+            "features": {
+                "VkPhysicalDeviceRelaxedLineRasterizationFeaturesIMG": {
+                    "relaxedLineRasterization": true
+                }
+            }
+        }
+    },
+    "profiles": {
+        "VP_ANDROID_15_v": {
```

Comment:
I would recommand a more explicit name for the profile such as `VP_ANDROID_15_v_requirements` to follow the precedent of Zink, DXVK and VKD3D-proton profiles that are also requirements profiles but naming this is up to you as far as I am concerned.

Reply:
We spoke internally, and the new name is `VP_ANDROID_15_minimums`.  This is because we may have a bigger-than-minimums extension in the future.

Reply:
Ok!